tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -752,7 +769,7 @@ No cleanup is necessary if the next test run is the [Database No Migration Test]
 
 To cleanup from this test you can use one of two scripts. The `sql/cleanup.sql`
 script will remove all objects including the *disaster_recovery* database. The
-`sql/orca_schema_v2/remove.sql` script will remove only the objects created
+`sql/orca_schema_v4/remove.sql` script will remove only the objects created
```

Comment:
This should be expanded. If you use `sql/orca_schema_v4/remove.sql` you will remove only the v4 migrations. You should specify that you should run them in reverse order to remove the migrations you wish to remove. i.e 
- to remove v4 migration run `sql/orca_schema_v4/remove.sql`
- to remove v3 migration run `sql/orca_schema_v4/remove.sql` then `sql/orca_schema_v3/remove.sql`
- to remove v2 migration run `sql/orca_schema_v4/remove.sql` then `sql/orca_schema_v3/remove.sql` then `sql/orca_schema_v2/remove.sql`

Reply:
added additional instructions

Reply:
Rework paragraph and provide more context so an end user understands how to rollback a specific migration for further testing.